### PR TITLE
chore: update test to version 0.27.1

### PR DESCRIPTION
**Pull request by krol3**

### CODE REVIEW

**Comment by knqyf263** [comment]
@krol3 What do you mean "update test"?

**Comment by knqyf263** [comment]
Looks like this PR updates Trivy version. It also updates tests though.

**Comment by krol3** [comment]
> @krol3 What do you mean "update test"?

Using bats, we have unit test for the entrypoint.sh. I updated with the option "secret" for example.

**Comment by knqyf263** [comment]
>Using bats, we have unit test for the entrypoint.sh. I updated with the option "secret" for example.

Hmm, so doesn't this PR update Trivy version? This change affects only tests, right?

**Comment by krol3** [comment]
> > Using bats, we have unit test for the entrypoint.sh. I updated with the option "secret" for example.
> 
> Hmm, so doesn't this PR update Trivy version? This change affects only tests, right?

@knqyf263  I update the image version here https://github.com/krol3/trivy-action/blob/trivy-0.27/Dockerfile#L1
and also the test of using this version here: https://github.com/krol3/trivy-action/blob/trivy-0.27/test/test.bats#L40

**Comment by knqyf263** [comment]
If we update the Trivy version, I'm feeling like the test is not main concern in this PR. Am I correct? I saw the PR title and commit message, then I thought this PR just updated tests. But it looks like it updates the Trivy version as well. That's why I'm confused as I'm not so familiar with trivy-action.

>@knqyf263 I update the image version here https://github.com/krol3/trivy-action/blob/trivy-0.27/Dockerfile#L1

You mean this action will use v0.27.0, right?

File: test/test.bats [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
 @test "trivy fs with securityChecks option" {
   # trivy fs -f json --security-checks=vuln,config -o fs.test .
-  ./entrypoint.sh '-a fs' '-j .' '-b json' '-s vuln,config' '-h fs-scheck.test'
+  ./entrypoint.sh '-a fs' '-j .' '-b json' '-s vuln,config,secret' '-h fs-scheck.test'
```
Comment (simar7) [inline test/test.bats:40]: 1. Where is `fs-scheck.test`? Did you forgot to check it in?
2. Can we update the repo with a dummy secret and accordingly update the golden file to make an assertion?

**Comment by simar7** [comment]
hi @krol3 - any update on this? Some users are trying to use secret scanning with Trivy Action and won't be able to until we update it with merging this PR.

File: test/test.bats (hunk 1 above)
Comment (krol3) [reply test/test.bats:40]: yeah! @simar7 I forgot to add the data/fs-scheck.test

Comment (krol3) [reply test/test.bats:40]: Related (2), I using the trivy repo with the secret option.

**Comment by krol3** [comment]
@simar7  the PR was updated with all your review's point

**Comment by b34rd-tek** [comment]
If there is anything I can help with on this PR, please let me know. I'm anxiously waiting for secret scanning and I don't want to build a custom action just for my use. :-)

**Review by simar7 — APPROVED** [review]

**Pull request merged**